packages/notifi-core/lib/operations/GetSupportedTargetTypesForDapp.ts

```diff
@@ -0,0 +1,7 @@
+import { ParameterLessOperation, TargetType } from '../models';
+
+export type GetSupportedTargetTypesResult = ReadonlyArray<TargetType>;
+
+export type GetSupportedTargetTypesForDappService = Readonly<{
+  getSupportedTargetTypesForDapp: ParameterLessOperation<GetSupportedTargetTypesResult>;
```

Comment:
i don't think we should use ParameterLessOperation, we should take a ClientGetSupportedTargetTypesForDappInput as an input right?

Reply:
yup, I missed that